test/translator/testData/iso19115-2-datagov-to-dcatus.json

```diff
@@ -15,8 +15,6 @@
   },
   "identifier": "ISO19115-2-ID-123456",
   "accessLevel": "non-public",
-  "bureauCode": [],
-  "programCode": [],
```

Comment:
Note: because of this, all DCAT-US items translated from ISO should be validated under the non-federal schema. This isn't a bad thing, in fact it is expected, just making sure that's what we're doing.

Reply:
Good callout. I noticed the same on the other end: https://github.com/GSA/datagov-harvester/blob/main/harvester/harvest.py#L106-L113. 

Pretty sure all our XML sources would fail federal validation, but I'd argue that the best practice would be to generate a separate `iso-to-dcat-transform.json` with looser validation than both.